README.md

```diff
@@ -34,7 +34,7 @@ $ yarn
 4. Start development server:
 
 ```sh
-$ make dev
+$ yarn dev
```

Comment:
Probably worth changing the `make dev` command in the `makefile` itself, even if we're just calling `yarn dev`

```make
dev: clear install
	$(call banner, "gateway is starting")
	$(call log, "starting development server")
	@yarn dev
```